Source/OxyPlot/Rendering/RenderContext/RenderContextBase.cs

```diff
@@ -26,152 +26,120 @@ protected RenderContextBase()
         }
 
         /// <summary>
-        /// Gets or sets a value indicating whether the context renders to screen.
+        /// Gets a value indicating whether the specified points form a straight line (i.e. parallel to the pixel raster).
         /// </summary>
-        /// <value><c>true</c> if the context renders to screen; otherwise, <c>false</c>.</value>
-        public bool RendersToScreen { get; set; }
+        /// <param name="p1">The first point.</param>
+        /// <param name="p2">The second point.</param>
+        /// <returns>true if the points form a straight line; false otherwise.</returns>
+        public static bool IsStraightLine(ScreenPoint p1, ScreenPoint p2)
+        {
+            const double epsilon = 1e-5;
+            return Math.Abs(p1.X - p2.X) < epsilon || Math.Abs(p1.Y - p2.Y) < epsilon;
```

Comment:
No there is no precedent for this, the 1e-5 is pretty much arbitrary.

Your concern is of course valid, even though there is no such renderer at the moment. But new renderer implementations don't necessarily need to use this function - it is only used by the `RenderContextBase.ShouldDraw[xxx]Aliased(...)` functions, and those are in fact virtual, so it would be easy to insert some custom logic there.

On the other hand, maybe it would be sensible to expose the `epsilon` as a function parameter with 1e-5 as a default value? This would also contribute to the 'self-documenting' nature of this function.

Reply:
> those are in fact virtual

Good point. So long as this isn't used by any non-virtual code I think it's fine. I'd be in favour of adding some more detail on the implementation into the documentation, so that if other implementations do use it, they'll know they can depend on that particular behaviour.

I don't think making `epsilon` a parameter is necessary.